registry/storage/driver/s3-aws/s3_test.go

```diff
@@ -23,13 +21,8 @@ import (
 	"github.com/distribution/distribution/v3/registry/storage/driver/testsuites"
 )
 
-// Hook up gocheck into the "go test" runner.
-func Test(t *testing.T) { check.TestingT(t) }
-
-var (
-	s3DriverConstructor func(rootDirectory, storageClass string) (*Driver, error)
-	skipS3              func() string
-)
+var s3DriverConstructor func(rootDirectory, storageClass string) (*Driver, error)
+var skipS3 func() string
```

Comment:
Any reason why these are no longer grouped?

Reply:
I dont have a preference per se, just noticed it in the diff and was curious about why